examples/step-66/doc/results.dox

```diff
@@ -0,0 +1,150 @@
+
+In any case, there is one more very useful optimization that should be mentioned here. In the <code>compute_update()</code> function we explicitly reset the vector holding the Newton update before passing as output vector to the SolverCG. In that case we use a zero staring value for the CG method, which is more suitable than the previous Newton step and thus reduces the number of CG iterations by some few steps.
+
+TODO Mathematical background of this? Zero vector is in every subspace contained.
+
+
+
+<h3>Possibilities for extensions</h3>
+
+A couple of possible extensions are available.
+
+<h4>More sophisticated Newton iteration</h4>
+Beside a step size controlled version of the Newton iteration, one could implement a more flexible stopping criterion for the Newton iteration. For example based on a mixed error control with an given absolute and relative tolerance.
+
+TODO Citation?
+
+<h4>Eigenvalue problem</h4>
+One can consider the corresponding eigenvalue problem, which is called Bratu problem. For example, if we define a fixed eigenvalue $\lambda\in[0,6]$, we can compute the corresponding discrete eigenfunction. You will notice that the number of Newton steps will increase with increasing $\lambda$. To reduce the number of Newton steps you can use a trick by starting from a certain $\lambda$, compute the eigenfunction, increase $\lambda=\lambda + \delta_\lambda$ and use the previous solution as initial guess for the Newton iteration. In the end you can plot the $H^1(\Omega)$-norm over the eigenvalue. What do you observe for further increasing $\lambda>7$?
+
+
+
+
+<h4>Extension for future update of the tutorial</h4>
+
+- Parallel scalability, see castelli2021numerical
+
+- Comparison against matrix-based AMG, see step-50
+
+- Thread parallelism
+
+- @kronbichler's comment in #8229
+Can you make MGTransferMatrixFree a class member? From an educational point of
+view, I think it would make sense to show that the transfer only needs to be
+set up once and can be re-used in subsequent nonlinear iterations. This is
+different to the mg_smoother object that must indeed be rebuilt because the
+diagonal changes (even though one could try to find a balance of not
+recomputing in every iteration - that would make for a good possibility for
+extensions, though).
```

Comment:
I think this comment can be removed since it is already a class member.

Reply:
OK, I will open an issue: Step-66 general update.

@peterrum it was more about the second part of @kronbichler's comment. As I understood it right maybe this would yield some form of a chord Newton method, where the Jacobian (and thus the preconditioner) is only build on demand, e.g. if convergence is not reached in less the some few steps.